tools/ansible/playbooks/bootstrap.yml

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+---
+- hosts: all
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  roles:
+    - dlpx.appliance-build-bootstrap
```

Comment:
I'd rather adopt a name like `appliance-build.bootstrap`, which doesn't have "dlpx" in the name, and is more consistent with the naming of other roles in this repository.